shell/common/rasterizer.cc

```diff
@@ -267,6 +269,15 @@ RasterStatus Rasterizer::DoDraw(
   timing.Set(FrameTiming::kRasterFinish, fml::TimePoint::Now());
   delegate_.OnFrameRasterized(timing);
 
+  const fml::TimePoint latest_frame_target_time =
+      delegate_.GetLatestFrameTargetTime();
+  const fml::TimePoint frame_target_time = layer_tree->target_time();
+  if (latest_frame_target_time > frame_target_time) {
+    fml::tracing::TraceEventAsyncComplete("flutter", "SceneDisplayLag",
```

Comment:
I thought we had macros for these. Should we even bother with the macros elsewhere? Those seem like a relic from a time where they were defined away. We don't do that anymore.

Reply:
There seem to be one variant of macros that exist but are slightly orthogonal to this which are: `TRACE_EVENT_ASYNC_BEGIN{0, 1}`, they take in `id` as opposed to `begin_time` and `end_time`. I was able to find only one use of these in `animator.cc` for `Frame Request Pending` event. I think it will be nice to unify on macros or function wrapper but as it stands both these variants are used.

I think we should file an issue to track this unification (should be fairly small but i'd like to keep that out of this PR). WDYT?

Reply:
> I think we should file an issue to track this unification (should be fairly small but i'd like to keep that out of this PR). WDYT?

Sounds good.